python/cuspatial/cuspatial/tests/test_geoseries.py

```diff
@@ -621,3 +622,97 @@ def test_reset_index(level, drop, name, inplace):
         pd.testing.assert_series_equal(expected, got.to_pandas())
     else:
         pd.testing.assert_frame_equal(expected, got.to_pandas())
+
+
+def test_geocolumn_polygon_accessor(naturalearth_lowres):
```

Comment:
Looks like you forgot to add this fixture.

Reply:
Removed, this is not needed.